test/lib/tasks/seed_rake_test.rb

```diff
@@ -16,12 +16,12 @@ def self.glob_dummy_seeds
   end
 
   describe 'seeds with dependency' do
-    subject { Rake.application.tasks_in_scope(defined?(Rake::Scope) ? Rake::Scope.new('db:seed') : %w(db seed)) }
+    subject { Rake.application.tasks_in_scope(defined?(Rake::Scope) ? Rake::Scope.new('db:seed') : %w[db seed]) }
```

Comment:
I'm on board with this. Especially since Hound and Rubocop have differing defaults.